src/psqt.cpp

```diff
@@ -782,11 +782,6 @@ void init() {
           for (Square s = SQ_A1; s <= SQ_H8; ++s)
           {
               File f = std::min(file_of(s), FILE_H - file_of(s));
-#ifdef RACE
-              if (var == RACE_VARIANT)
-                  psq[var][ pc][ s] = v + Bonus[var][pc][rank_of(~s)][f] + make_score(10 * rank_of(s), 10 * rank_of(s));
-              else
-#endif
               psq[var][ pc][ s] = v + Bonus[var][pc][rank_of(s)][f];
 #ifdef RACE
```

Comment:
Yes. This is the essential part, because `~s` is the symmetry we do not want to use in racing kings. In principle, I think we could simplify away `horizontal_flip` (`horizontal_flip(s)` -> `s`) because of line 784, but it explicitly describes what the code is intended to do.